packages/dialtone-vue3/recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -365,26 +373,20 @@ export default {
     right: var(--dt-space-450);
   }
 
-  .content-text-wrapper-class:not(img) {
-    line-height: 1.6rem;
-  }
-
   &:deep(.dt-item-layout) {
     font: var(--dt-typography-body-base);
     min-height: initial;
     padding: 0px;
-    .d-avatar {
-      margin-top: var(--dt-space-300);
-    }
   }
 
   &:deep(.dt-item-layout--left) {
     align-self: flex-start;
     text-align: end;
     display: block;
-    .d-avatar {
-      margin-top: var(--dt-space-300);
-    }
+    padding-right: var(--dt-space-300);
+    padding-left: var(--dt-space-0);
+    // min-width = avatar width + padding
```

Comment:
Would change it to /* min-width = avatar width + padding */
It could get confused if it is with `//`